src/data_nodes.cpp

```diff
@@ -88,6 +91,12 @@ static DataNode data_nodes[] = {
     TS_NODE_UINT32(0x20, "Timestamp_s", &timestamp,
         ID_INFO, TS_ANY_R | TS_ANY_W, PUB_SER | PUB_NVM),
 
+    TS_NODE_UINT32(0x21, "FlashSize_kb", &flash_size,
+        ID_INFO, TS_ANY_R | TS_ANY_W, PUB_SER | PUB_NVM),
+
+    TS_NODE_UINT32(0x22, "FlashPageSize_kb", &page_size,
+        ID_INFO, TS_ANY_R | TS_ANY_W, PUB_SER | PUB_NVM),
+
```

Comment:
The correct unit is KiB and those values should not be published to the serial by default or stored in the NVM as they are constant anyway. Also configure them as read-only.
```suggestion
    TS_NODE_UINT32(0x21, "FlashSize_KiB", &flash_size,
        ID_INFO, TS_ANY_R, 0),

    TS_NODE_UINT32(0x22, "FlashPageSize_KiB", &page_size,
        ID_INFO, TS_ANY_R, 0),
```